lib/utils.js

```diff
@@ -6,6 +6,7 @@ const resolvePath = require('path').resolve
 const once = require('ramda').once
 const walkBack = require('walk-back')
 const chalk = require('chalk')
+const delim=(require('os').platform()==='win32')? '\\' :'/'
```

Comment:
This line is breaking standard style. I guess it should be like this

```javascript
const delim = (require('os').platform() === 'win32') ? '\\' :'/'
```

Please check it.